### PR TITLE
Setup e2e testing

### PR DESCRIPTION
**Pull request by pradel**
The goal of this pr is to implement e2e for the suite. I am trying an approach that will allow us to write generic tests that should work no matter which database or transport is used.
I am using the accounts client in order to do the requests.
Any feedback welcome 🙌

### CODE REVIEW

File: packages/e2e/__tests__/servers/server.ts [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import * as mongoose from 'mongoose';
+import { ApolloServer, gql } from 'apollo-server';
+import { merge } from 'lodash';
+import fetch from 'node-fetch';
+
+// Server
+import { AccountsServer } from '@accounts/server';
+import { AccountsPassword } from '@accounts/password';
+// TODO rename to AccountsMongo ?
+import { Mongo } from '@accounts/mongo';
```
Comment (TimMikeladze) [inline packages/e2e/__tests__/servers/server.ts]: This server instance is tied to mongo at the moment. If the database is parameterized then we could test all the database drivers using the same set of tests.

Comment (pradel) [reply packages/e2e/__tests__/servers/server.ts]: done

**Comment by pradel** [comment]
Ready for review.
The tests are run against the password module only for now.
Tests for the `rest` and `graphql` server using the mongo database.

**Comment by pradel** [comment]
They will help us catch the regressions that we might introduce in the future.
More tests can be added on separate pull requests.

**Comment by TimMikeladze** [comment]
@pradel Could you please resend this PR? I accidentally merged this branch. There's a reverted commit now on master in order to fix this. Sorry.

**Pull request merged**